x-pack/platform/packages/shared/onechat/onechat-common/base/errors.ts

```diff
@@ -184,6 +185,33 @@ export const createRequestAbortedError = (
   return new OnechatError(OnechatErrorCode.requestAborted, message, meta ?? {});
 };
 
+/*
+ * Produce a string version of an error,
+ */
+export function formatOnechatErrorMessage(error: any): string {
```

Comment:
Should be in `@kbn/onechat-browser` and not `@kbn/onechat-common` I think, it seems like we would only ever use that on the browser-side? ( `error.body` is only from fetch errors, and we don't `inspect` on the server)